api/handler/seattle.go

```diff
@@ -127,10 +127,10 @@ func (h *Handler) seattleGetOfficerByBadgeHistorical(badge string, w http.Respon
 	}
 
 	sort.Slice(officers, func(a, b int) bool {
-		if officers[a].LastName == officers[b].LastName {
-			return officers[a].FirstName < officers[b].FirstName
+		if officers[a].Date == officers[b].Date {
+			return officers[a].Date > officers[b].Date
 		}
-		return officers[a].LastName < officers[b].LastName
+		return officers[a].Date > officers[b].Date
```

Comment:
What's this change for?

Reply:
I was sorting historical returns by officer name rather than by the date for some reason. I'm not exactly sure why I was doing that, it was probably a bad copy/paste. This basically sorts the returns in reverse chronological order.

Reply:
Oh I totally missed this was in the historical search function, got it!